bot/models/migrations/006_up__selectable_roles.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS selectable_roles (
+    guild_id           BIGINT NOT NULL,
+    role_id            BIGINT NOT NULL,
+    role_name          VARCHAR NOT NULL,
```

Comment:
Should be a foreign key to custom_roles (role_id), see persisted_roles for example.
When this is done you also don't need to keep role_name here, it'll be stored in custom_roles table